delta/app/src/main/resources/app.conf

```diff
@@ -254,7 +254,7 @@ app {
       # the default number of results if none is specified
       default-size = 30
       # the maximum number of results if none is specified
-      size-limit = 100
+      size-limit = 1000
```

Comment:
Why putting a value that large by default ? v1.4 only set 100

Reply:
It was discussed in a call... Fusion uses `1000`. I just checked, v1.4 default was `100` so I think the service silently ignored larger inputs.
@umbreak, @dhaneshnm thoughts on what we should about this?

Reply:
In the deployment configurations we have it much larger. 1000 as limit I find alright.

Reply:
I don't see any problem with a max of 1000

Reply:
Can it be a problem for blazegraph ? For elasticsearch, it should be ok but it could put more (too much ?) pressure on BG ?
If we change it, we should not forget to document it in the changes

Reply:
@imsdu, do you have any concerns with a max value of 1000?

Reply:
> Can it be a problem for blazegraph ?

Blazegraph queries do not use this pagination... the start and limits are set in the query itself.

Reply:
We can always override this value for blazegraph if we want to.

Reply:
so, ok to merge?